Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:xenial
+LABEL maintainer="Shaun Jackman <sjackman@gmail.com>"
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends bzip2 ca-certificates curl file fonts-dejavu-core g++ git locales make openssh-client patch sudo uuid-runtime \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN localedef -i en_US -f UTF-8 en_US.UTF-8 \
+	&& useradd -m -s /bin/bash linuxbrew \
+	&& echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
```

Comment:
Is this standard practise for Docker?

Reply:
I didn't see it recommended on the official docker website, but I see something similar here:

* http://gbraad.nl/blog/non-root-user-inside-a-docker-container.html

Reply:
Standard practice for Docker is to leave the user as `root`, but `brew` refuses to run as root.

Minor style point, I can instead put this file in `/etc/sudoers.d/`. Any preference?

Reply:
> Minor style point, I can instead put this file in `/etc/sudoers.d/`. Any preference?

No preference from me; I defer.